File: spring-rest-docs/src/main/java/com/example/restdocs/member/MemberController.java
package com.example.restdocs.member;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/members")
@RequiredArgsConstructor
public class MemberController {
    
    private final MemberService memberService;
    
    @GetMapping("/{id}")
    public Member getMember(@PathVariable Long id){
        return memberService.getMember(id);
    }
}

File: kotlin-multi-module/api/src/main/kotlin/com/tls/boot/api/MemberController.kt
package com.tls.boot.api

import com.tls.boot.core.Member
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/members")
class MemberController {

    @GetMapping
    fun getMember():Member{
        return Member("test",123)
    }
}